Extension/src/LanguageServer/client.ts

```diff
@@ -1392,6 +1400,8 @@ export class DefaultClient implements Client {
         if (document.uri.scheme === "file") {
             openFileVersions.set(document.uri.toString(), document.version);
         }
+        // Log warm start.
+        clientCollection.timeTelemetryCollector.setActivationTime(document.uri.fsPath);
```

Comment:
I'm going to diverge from the table we created and suggest that we call this "didOpen" time.  Activation means something else and only happens once, so I think it is confusing to refer to this as "activation" time.